CHANGELOG.md

```diff
@@ -2,6 +2,69 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x will End of Life on September 8th, 2021.** Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) to plan your migration to the latest 2.x version. Support for the 1.x version ended on December 4th, 2020.
 
+2.17.0 (In Progress)
+====================
+**Preflight API Pilot**
```

Comment:
We already have this in beta in an older version right? Should this be beta instead? cc @seancoleman2

Reply:
I do not think we want to release this as "beta" release - since there is additional cost of keeping it from the mainline, and then updating it everytime mainline progresses. But we can decide on those details closer to releasing to public.